ladybug/skymodel.py

```diff
@@ -216,3 +291,880 @@ def sky_temperature(horiz_ir, dry_bulb):
+        [1.000000, 0.746140, 0.751740, 0.598390, 0.725230],
+        [0.922210, 0.500000, 0.376800, 0.517110, 0.548630],
+        [0.500000, 0.450000, 0.429970, 0.404490, 0.539940],
+        [0.960430, 0.881630, 0.775640, 0.596350, 0.937680]]
+
+    coeffs[5][0] = [
+        [1.030000, 1.040000, 1.000000, 1.000000, 1.049510],
+        [1.050000, 0.990000, 0.990000, 0.950000, 0.996530],
+        [1.050000, 0.990000, 0.990000, 0.820000, 0.971940],
+        [1.050000, 0.790000, 0.880000, 0.820000, 0.951840],
+        [1.000000, 0.530000, 0.440000, 0.710000, 0.928730],
+        [0.540000, 0.470000, 0.500000, 0.550000, 0.773950],
+        [1.038270, 0.920180, 0.910930, 0.821140, 1.034560]]
+
+    coeffs[5][1] = [
+        [1.041020, 0.997520, 0.961600, 1.000000, 1.035780],
+        [0.948030, 0.980000, 0.900000, 0.950360, 0.977460],
+        [0.950000, 0.977250, 0.869270, 0.800000, 0.951680],
+        [0.951870, 0.850000, 0.748770, 0.700000, 0.883850],
+        [0.900000, 0.823190, 0.727450, 0.600000, 0.839870],
+        [0.850000, 0.805020, 0.692310, 0.500000, 0.788410],
+        [1.010090, 0.895270, 0.773030, 0.816280, 1.011680]]
+
+    coeffs[5][2] = [
+        [1.022450, 1.004600, 0.983650, 1.000000, 1.032940],
+        [0.943960, 0.999240, 0.983920, 0.905990, 0.978150],
+        [0.936240, 0.946480, 0.850000, 0.850000, 0.930320],
+        [0.816420, 0.885000, 0.644950, 0.817650, 0.865310],
+        [0.742960, 0.765690, 0.561520, 0.700000, 0.827140],
+        [0.643870, 0.596710, 0.474460, 0.600000, 0.651200],
+        [0.971740, 0.940560, 0.714880, 0.864380, 1.001650]]
+
+    coeffs[5][3] = [
+        [0.995260, 0.977010, 1.000000, 1.000000, 1.035250],
+        [0.939810, 0.975250, 0.939980, 0.950000, 0.982550],
+        [0.876870, 0.879440, 0.850000, 0.900000, 0.917810],
+        [0.873480, 0.873450, 0.751470, 0.850000, 0.863040],
+        [0.761470, 0.702360, 0.638770, 0.750000, 0.783120],
+        [0.734080, 0.650000, 0.600000, 0.650000, 0.715660],
+        [0.942160, 0.919100, 0.770340, 0.731170, 0.995180]]
+
+    coeffs[5][4] = [
+        [0.952560, 0.916780, 0.920000, 0.900000, 1.005880],
+        [0.928620, 0.994420, 0.900000, 0.900000, 0.983720],
+        [0.913070, 0.850000, 0.850000, 0.800000, 0.924280],
+        [0.868090, 0.807170, 0.823550, 0.600000, 0.844520],
+        [0.769570, 0.719870, 0.650000, 0.550000, 0.733500],
+        [0.580250, 0.650000, 0.600000, 0.500000, 0.628850],
+        [0.904770, 0.852650, 0.708370, 0.493730, 0.949030]]
+
+    coeffs[5][5] = [
+        [0.911970, 0.800000, 0.800000, 0.800000, 0.956320],
+        [0.912620, 0.682610, 0.750000, 0.700000, 0.950110],
+        [0.653450, 0.659330, 0.700000, 0.600000, 0.856110],
+        [0.648440, 0.600000, 0.641120, 0.500000, 0.695780],
+        [0.570000, 0.550000, 0.598800, 0.400000, 0.560150],
+        [0.475230, 0.500000, 0.518640, 0.339970, 0.520230],
+        [0.743440, 0.592190, 0.603060, 0.316930, 0.794390]]
+
+    return coeffs
```

Comment:
This is way too many numbers and the only way to ensure they will be working fine is to write tests and compare the results against the original library.

Reply:
Does this mean that you actually want me to add `pip install pvlib` to the Travis run?  Or is it ok if I just run their model locally, get some results, and write the test to compare it against that?

Reply:
>  Or is it ok if I just run their model locally, get some results, and write the test to compare it against that?

That's what I meant.

Reply:
I added a bunch of test that parallel pvlib's own tests:
https://github.com/pvlib/pvlib-python/blob/master/pvlib/test/test_irradiance.py

In the process, I also found what is clearly a bug in pvlib's code and I reported it on their github:
https://github.com/pvlib/pvlib-python/issues/637